src/app/tests/suites/certification/Test_TC_MF_1_4.yaml

```diff
@@ -28,20 +28,6 @@ config:
         defaultValue: "MT:-24J0AFN00KA0648G00" # This value needs to be generated automatically
 
 tests:
-    - label: "Stop target device"
-      cluster: "SystemCommands"
-      command: "Stop"
-
-    - label:
-          "Start target device with the provided discriminator for basic
-          commissioning advertisement"
-      cluster: "SystemCommands"
-      command: "Start"
-      arguments:
-          values:
-              - name: "discriminator"
-                value: discriminator
-
     - label: "TH_CR1 starts a commissioning process with DUT_CE"
```

Comment:
Why not adding a prompt in the MF test cases ?

Reply:
@raju-apple as per the MF testplan we don't need need reboot command